styles/_forms.scss

```diff
@@ -68,6 +68,11 @@
   margin-left: 2px;
 }
 
+.form-field .optional {
+  color: #bbb;
```

Comment:
not sure if this should be `$border-color`?

https://github.com/zendesk/copenhagen_theme/blob/master/styles/_variables.scss#L23

Reply:
why should it? that's a different colour - `#bbb` not `#ddd`. and I'd say it's pretty hard to read the text when using lighter one.

Reply:
Argh, sorry I am not reading properly today. You're right! I thought I read `$border-color: #bbb` 🙈

Reply:
Does this mean that the text for 'optional' will be #BBB? If so, it won't have sufficient contrast to meet the accessibility requirement.

According to https://webaim.org/resources/contrastchecker/, #BBB against a white background gives a contrast of 1.92. For text of this size, it would need to have a contrast of at least 4.5

Reply:
Thanks for bringing this! It somehow fell out of my sight. It's indeed painful to read it with `#bbb`.
We could use `#68737D` from Garden here, its the closest one passing the 4.5 limit: 
![screen shot 2018-10-11 at 11 50 51](https://user-images.githubusercontent.com/1819404/46796254-82c5a680-cd4c-11e8-90ed-88baeab70e53.png)
Though in this case it is almost indistinguishable from label text. How do you think?
@milenaalarsen

Reply:
From our typography guidelines, we are encouraged to use #2F3941 for primary/ standard and #68737D for secondary. 

When a field is disabled and the user can't write inside the form, the font color inside is #C2C8CC. But we don't have anything in place for disable titles.  Does this help?

Reply:
Thank you @augustocravosilva, I didn't know that! I think we should go with `$secondary-text-color`. And maybe we should consider switching Help Center to use proposed Garden colours but that's out of scope for this PR. I'll update the colour in all forms.

Reply:
Sounds good :)
Please run `bin/compile.rb` to produce the final CSS